src/worldcereal/openeo/feature_extractor.py

```diff
@@ -0,0 +1,111 @@
+"""Feature computer GFMAP compatible to compute Presto embeddings."""
+
+import xarray as xr
+from openeo.udf import XarrayDataCube
+from openeo_gfmap.features.feature_extractor import PatchFeatureExtractor
+
+
+class PrestoFeatureExtractor(PatchFeatureExtractor):
+    """Feature extractor to use Presto model to compute per-pixel embeddings.
+    This will generate a datacube with 128 bands, each band representing a
+    feature from the Presto model.
+
+    Interesting UDF parameters:
+    - presto_url: A public URL to the Presto model file. A default Presto
+      version is provided if the parameter is left undefined.
+    - rescale_s1: Is specifically disabled by default, as the presto
+      dependencies already take care of the backscatter decompression. If
+      specified, should be set as `False`.
+    """
+
+    PRESTO_MODEL_URL = "https://artifactory.vgt.vito.be/artifactory/auxdata-public/worldcereal-minimal-inference/presto.pt"  # NOQA
+    PRESO_WHL_URL = "https://artifactory.vgt.vito.be/artifactory/auxdata-public/worldcereal/dependencies/presto_worldcereal-0.1.0-temp-py3-none-any.whl"
+    BASE_URL = "https://s3.waw3-1.cloudferro.com/swift/v1/project_dependencies"  # NOQA
+    DEPENDENCY_NAME = "worldcereal_deps.zip"
+
+    GFMAP_BAND_MAPPING = {
```

Comment:
This is in fact a bit an overhead, if later on in `presto-worldcereal` we again remap: https://github.com/WorldCereal/presto-worldcereal/blob/main/presto/inference.py#L41

So we should consider to immediately use the presto naming here and remove the remapping in `presto-worldcereal`